record_formatter_test.go

```diff
@@ -141,6 +143,95 @@ func TestDebeziumConverter(t *testing.T) {
 	is.Equal(got, want)
 }
 
+func TestDebeziumConverter_RawData(t *testing.T) {
+	is := is.New(t)
+	converter, err := DebeziumConverter{}.Configure(map[string]string{})
+	is.NoErr(err)
+
+	r := Record{
+		Position:  Position("foo"),
+		Operation: OperationCreate,
+		Metadata:  Metadata{MetadataConduitSourcePluginName: "example"},
+		Key:       RawData("bar"),
+		Payload: Change{
+			Before: RawData("foo"),
+			After:  nil,
```

Comment:
should we maybe test both `Before` and `After` together?

Reply:
Can you remind me, what happens if key is raw data too?

Reply:
The key doesn't make a difference, the debezium record only contains the payload. The kafka connector produces a kafka connect key if it sees that `sdk.record.format` is set to `debezium`.